ilastik/utility/slot_name_enum.py

```diff
@@ -0,0 +1,53 @@
+import enum
+from typing import List, Tuple
+from itertools import chain
+
+class SlotNameEnum(enum.IntEnum):
```

Comment:
I don't know if this class should be extended, or one should think of something else, but in principle there are two types of data roles: 1) required (`raw_data`...), optional (`mask`...). Would be cool to represent this somehow

Reply:
We can eventually push that stuff into the opDataSelection, but that'll have to wait for the next release =/